src/main/java/nl/ordina/jobcrawler/controller/ScraperController.java

```diff
@@ -1,18 +1,12 @@
 package nl.ordina.jobcrawler.controller;
 
-
+import nl.ordina.jobcrawler.service.ScraperService;
 import nl.ordina.jobcrawler.service.ScraperService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-
-/*
- * This controller contains the endpoint to start scraping
-*/
-
-
 @CrossOrigin
 @RestController
 public class ScraperController {
```

Comment:
why cant we add @Autowired directly to the object. why we are creating a class constructor for this?

Reply:
As a former member Gijs of our team pointed out, the autowire is better on the constructor because it gives more control over possible exceptions when the bean is constructed.

Reply:
From a quick search, adding the ```@Autowired``` directly to the attributes is called **Field injection** while adding it to the constructor is called **Constructor injection**. 
There are many articles and blog posts about which to use, and all of them recommend using **Constructor injection**. 
[This article](https://blog.marcnuri.com/field-injection-is-not-recommended/) is one of many I was able to find.